sodym/export/data_writer.py

```diff
@@ -1,139 +1,64 @@
 import logging
 import os
 import pickle
-from typing import Optional
-
-import numpy as np
-import plotly.graph_objects as go
-from pydantic import BaseModel as PydanticBaseModel
+from typing import Callable
 
 from ..mfa_system import MFASystem
+from ..named_dim_arrays import NamedDimArray
 from .helper import to_valid_file_name
 
 
-class DataWriter(PydanticBaseModel):
-
-    sankey: dict = {'do_visualize': True, 'color_scheme': 'blueish'}
-    display_names: Optional[dict] = {}
-
-    def visualize_results(self, mfa: MFASystem):
-        if self.sankey['do_visualize']:
-            self.visualize_sankey(mfa=mfa)
-
-    def export_mfa_to_pickle(self, mfa: MFASystem, export_path: str):
-        dict_out = self._convert_to_dict(mfa)
-        pickle.dump(dict_out, open(export_path, "wb"))
-        logging.info(f'Data saved to {export_path}')
-
-    def export_mfa_flows_to_csv(self, mfa: MFASystem, export_directory: str):
-        if not os.path.exists(export_directory):
-            os.makedirs(export_directory)
-        for flow_name, flow in mfa.flows.items():
-            path_out = os.path.join(export_directory, f'{to_valid_file_name(flow_name)}.csv')
-            flow.to_df().to_csv(path_out)
-        logging.info(f'Data saved in directory {export_directory}')
-
-    def export_mfa_stocks_to_csv(self, mfa: MFASystem, export_directory: str):
-        if not os.path.exists(export_directory):
-            os.makedirs(export_directory)
-        for stock_name, stock in mfa.stocks.items():
-            df = stock.stock.to_df()
-            path_out = os.path.join(export_directory, f'{to_valid_file_name(stock_name)}_stock.csv')
+def export_mfa_to_pickle(mfa, export_path: str):
+    dict_out = convert_to_dict(mfa)
+    pickle.dump(dict_out, open(export_path, "wb"))
+    logging.info(f'Data saved to {export_path}')
+
```

Comment:
Very picky pep-y comment: top level functions (not class methods), should be surrounded by two blank lines.